MAINTAINERS.md

```diff
@@ -0,0 +1,66 @@
+
+## General governance
+
+The SPIRE Helm CHarts project abides by the same [governance procedures][1] as the SPIFFE project, and ultimately reports to the [SPIFFE Steering Committee][3] (SSC) the same way that the SPIFFE project and associated maintainers do.
+
+SSC members do not track day-to-day activity in the SPIFFE/SPIRE projects, and this should be considered when deciding to raise issues to them. While the SSC has the ultimate say, in practice they are only engaged upon serious maintainer disagreement.
+
+## Maintainer responsibility
+
+SPIRE maintainers adhere to the [requirements and responsibilities][2] set forth in the SPIFFE governance document. They further pledge the following:
+
+* To act in the best interest of the project at all times.
+* To ensure that project development and direction is a function of community needs.
+* To never take any action while hesitant that it is the right action to take.
+* To fulfill the responsibilities outlined in this document and its dependents.
+
+## Guiding principles
+
+The goal of this project is to deliver Helm Charts to the user that are stable, secure, and simple.
+
+1. **Stability** -- The focus of this project is on stability over rapid feature development. SPIRE is a key infrastructure component that needs to be stable. While it is important to develop new features regularly, maintainers must take care that the project is stable.
+1. **Security** -- SPIRE is a security solution. Maintainers will ensure that the helm charts are secure along with guidance on increasing security for production environments.
+1. **Simplicity** -- Minimizing the options in the `values.yaml` files. Too many options can be confusing for the end user and we should limit options and to only expose configurations that are necessary. Its not possible to support every use case so we will select the use cases that are most important to the community.
+
+It's easy to appreciate a technically advanced feature. It's harder to appreciate the absence of bugs, the slow but steady improvement in stability, or the reliability of a release process. But those things distinguish a good project from a great one.
+
+## Feature development
+
+Maintainers will maintain [milestones](https://github.com/spiffe/helm-charts/milestones) in GitHub as the roadmap to direct feature development. Maintainers will factor in open issues, PRs from the community, and their own judgement to determine the what is priority for the project. Maintainers will discuss these issues at bi-weekly sync meetings, offline on slack, and through discussion on the various issues and PRs. After discussing all the factors, the maintainers will decide on the milestones.
+
+Maintainers will check in with each other before making changes to the milestones. Maintainers will focus development and review efforts towards issues and PRs added in milestones, with the closest milestone getting the highest priority.
```

Comment:
> Maintainers will focus development

If contributors that happen to be maintainers, can't focus on contributing the features they need, they can very well stop being both contributors and maintainers.

As far as I know, no one here is being paid to focus their development time exclusively on project priorities.